### DISCUSSION TITLE
How to avoid off auto opened/closed browser while using windowsdriver at desktop app.

### DISCUSSION THREAD

**Discussion by MrGultekin** [seleniumbase/SeleniumBase#2058] [Q&A] [answered]
I would like to use appium windows driver with seleniumbase, however browser opens and closes itself while executing tests. How to avoid it because i don't need to invoke any browser while testing desktop applications.

**Comment by mdmintz** [comment, marked as answer]
Check out the available syntax formats (currently there are 23): https://github.com/seleniumbase/SeleniumBase/blob/master/help_docs/syntax_formats.md
There's probably at least one format for what you're trying to accomplish. With many of them, the browser will automatically open at the start of tests and close automatically at the end of tests. There's also unlimited flexibility if you use one of the formats that involves overwriting the `get_new_driver()` method.
